### PR TITLE
feature/grants-3943

### PR DESCRIPTION
**Pull request by Hozan94**
This PR is for ticket:

- https://eaflood.atlassian.net/browse/GRANTS-3943

### CODE REVIEW

File: app/config/question-bank.js [hunk 1]
```diff
@@ -2939,7 +2948,7 @@ const questionBank = {
             building projects is more than £15,000 (40% of £37,500), you may still be eligible to apply for grant funding.
             `,
             insertText: '',
-            additionalPara: `If you are applying for grant funding for a single building, you can <a href="https://www.gov.uk/government/organisations/rural-payments-agency"> see other grants you may be eligible for.</a>`,
+            additionalPara: `If you are applying for grant funding for a single building, you can <a href="https://www.gov.uk/government/organisations/rural-payments-agency"> See other grants you may be eligible for</a>`,
```
Comment (zavies97) [inline app/config/question-bank.js:2951]: Why is the see capitalised? Looks like it isnt in line 2803 so not sure why it is here

File: app/config/question-bank.js [hunk 2]
```diff
@@ -3324,7 +3333,7 @@ const questionBank = {
             values: [{
               heading: 'Funding priorities',
               content: [{
-                para: `RPA want to fund multi-tier systems that have 3 tiers or fewer directly above each other, to reduce the risk of keel bone fractures from collisions and falls.`
+                para: `RPA want to fund aviary systems that have 3 tiers or fewer directly above each other, to reduce the risk of keel bone fractures from collisions and falls.`
```
Comment (zavies97) [inline app/config/question-bank.js:3336]: pretty sure this needs to be dynamic (multi-tier for hen, aviary for pullet)

File: app/config/question-bank.js (hunk 1 above)
Comment (Hozan94) [reply app/config/question-bank.js:2951]: @zavies97  good point, will revert it.

Comment (zavies97) [reply app/config/question-bank.js:2951]: Should this instead be **<a href="https://www.gov.uk/government/organisations/rural-payments-agency">see** (as this was the change made to the other additionalPara)?

Comment (Hozan94) [reply app/config/question-bank.js]: done

File: app/config/question-bank.js (hunk 2 above)
Comment (Hozan94) [reply app/config/question-bank.js]: done

File: app/helpers/handlers.js [hunk 3]
```diff
@@ -484,11 +484,11 @@ const getPage = async (question, request, h) => {
   let { url, nextUrlObject, type, title, hint, yarKey, ineligibleContent, label } = question
   const preValidationObject = question.preValidationObject ?? question.preValidationKeys 
   const nextUrl = getUrl(nextUrlObject, question.nextUrl, request)
-  const isRedirect = guardPage(request, preValidationObject, startPageUrl, serviceEndDate, serviceEndTime, ALL_QUESTIONS)
+  // const isRedirect = guardPage(request, preValidationObject, startPageUrl, serviceEndDate, serviceEndTime, ALL_QUESTIONS)
```
Comment (zavies97) [inline app/helpers/handlers.js:487]: This redirect needs to be commented back in (guessing it was commented out for testing?)

**Review by zavies97 — APPROVED** [review]

**Review by zavies97 — APPROVED** [review]

**Pull request merged**